data/lib/src/network/model/response/user_response.dart

```diff
@@ -31,13 +31,14 @@
 
 import 'package:data/src/util/attribute.dart';
 import 'package:domain/domain.dart';
+import 'package:equatable/equatable.dart';
 import 'package:json_annotation/json_annotation.dart';
 
-part 'user.g.dart';
+part 'user_response.g.dart';
 
 @JsonSerializable()
-class User {
-  User({
+class UserResponse extends Equatable {
```

Comment:
same, we should use `with EquatableMixin`

Reply:
Fixed check again pls